### PR TITLE
[infra-admin] Submit certificate request for abayer

### PR DESCRIPTION
**Pull request by abayer**

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Comment by MarkEWaite** [comment]
@dduportal and @lemeurherve will need to do this same type of step for Stephane Merle next week.

**Review by dduportal — APPROVED** [review]

**Comment by dduportal** [comment]
Waiting for @lemeurherve 's approval before merging, but LGTM

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**
